src/variables/colors.js

```diff
@@ -8,6 +8,8 @@ var colors = {
  oxfordBlue: '#333B4F',
  ebonyClay: '#293140',
  midnightExpress: '#1E2532',
+ blueBayoux:'#556285',
+ Fiord:'#3E4862',
```

Comment:
Here is where you need to change this to `fiord` the one in dist will be automatically generated